ebook/2_html2epub.sh

```diff
@@ -0,0 +1,41 @@
+# by Torben Menke https://entorb.net
+
+# run from within ebook dir via 
+# bash 2_html2epub.sh
```

Comment:
Does it require bash, or can any sh be used? It would be better if the latter!

Reply:
you are right, replaced by 
 `./2_html2epub.sh`